.pre-commit-config.yaml

```diff
@@ -13,33 +13,17 @@ repos:
   - id: mixed-line-ending
   - id: trailing-whitespace
 
-- repo: https://github.com/asottile/pyupgrade
-  rev: v3.3.1
-  hooks:
-  - id: pyupgrade
-    args: ["--py37-plus"]
-    exclude: ^cibuildwheel/resources/.*py$
-
-# Autoremoves unused imports
-- repo: https://github.com/hadialqattan/pycln
-  rev: v2.1.2
-  hooks:
-  - id: pycln
-    args: [--all]
-    stages: [manual]
-
-- repo: https://github.com/PyCQA/isort
-  rev: 5.11.4
-  hooks:
-  - id: isort
-    args: ["-a", "from __future__ import annotations"]
-    exclude: ^cibuildwheel/resources/.*py$
-
 - repo: https://github.com/psf/black
   rev: 22.12.0
   hooks:
   - id: black
 
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: v0.0.226
+  hooks:
+  - id: ruff
+    args: ["--force-exclude", "--fix"]
```

Comment:
The `--force-exclude` flag can be dropped in the next version of the action, https://github.com/charliermarsh/ruff-pre-commit/pull/20

Reply:
Wait 1-2 days and the next release will land.

Reply:
I'd expect so. :)

Reply:
Actually, it did, an hour ago. :)